### ISSUE TITLE
platform_browser_dynamic_1.platformBrowserDynamic is not a function

### ISSUE THREAD

**Issue by dheeraja00** [angular/angular#10732]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Upgraded to RC5 & getting this error: `platform_browser_dynamic_1.platformBrowserDynamic is not a function`, below is my main.ts, module.ts & component.ts

```
// main.ts
import {LocationStrategy, HashLocationStrategy} from '@angular/common';
import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
import {HTTP_PROVIDERS} from '@angular/http';
//import {enableProdMode} from '@angular/core';

import {APP_ROUTER_PROVIDERS} from './app/app.routes';
import {AppModule} from './app/app.module';

// enableProdMode()

platformBrowserDynamic().bootstrapModule(AppModule, [
  HTTP_PROVIDERS,
  APP_ROUTER_PROVIDERS,
  { provide: LocationStrategy, useClass: HashLocationStrategy }
])
.catch(err => console.error(err));
```
```
// module
import { NgModule }       from '@angular/core';
import { BrowserModule  } from '@angular/platform-browser';
import { App }   from './app';

@NgModule({
    declarations: [App],
    imports:      [BrowserModule],
    bootstrap:    [App],
})
export class AppModule {}
```
```
// component
import { Component } from '@angular/core';
import { Router, ActivatedRoute, ROUTER_DIRECTIVES, Event, NavigationEnd, NavigationStart } from '@angular/router';
declare let ga:Function;

@Component({
	selector: 'app',
	pipes: [],
	providers: [],
	directives: [ ROUTER_DIRECTIVES],
	styles: [String(require('../../../static/v4/angular/scss/app.scss'))],
	templateUrl: '../../../templates/v4/angular/app.html'
})
export class App {
	constructor(public router: Router) {
		ga('create', this._AnalyticsCode, {'cookieDomain': 'none'});

		this.router.events.subscribe((event:Event) => {
			if(event instanceof NavigationEnd) {
			    ga('send', 'pageview', event.urlAfterRedirects);
   	 		}
        });
	}
}
```

**Please tell us about your environment:**

* **Angular version:** 2.0.0-rc.5
 
* **Browser:** [all]

**Comment by pkozlowski-opensource** [comment]
Could you provide a minimal reproduction scenario in a plunker / share more info about your build? 

`platformBrowserDynamic` is well exported from `@angular/platform-browser-dynamic`, see: 
http://plnkr.co/edit/AYjxQ3iBO3tOG8ghV1d4

**Comment by pkozlowski-opensource** [comment]
Closing as we haven't heard back from the opener.

**Comment by eagor** [comment]
having the same issue with ver. 2.2.4;

All works just fine if `@angular/platform-browser-dynamic/bundles/platform-browser-dynamic.umd.js` is loaded directly to the browser.

But when trying to use webpack and es6 import:
`import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';`

I'm getting this in browser console:
`__WEBPACK_IMPORTED_MODULE_0__angular_platform_browser_dynamic__.a.platformBrowserDynamic is not a function`

**Comment by TheCraccer** [comment]
Apparently, we're having the same issue. Only happens on Android Phones with OS 4.4.4.

Angular Version 6.0.2

**Comment by vylesk** [comment]
I just had the same issue in a project I'm working on. We're using Angular 5.0.2 and Webpack on Windows in an IIS/ASP.NET project, with our primary browser as Chrome.
My issue randomly appeared after rebooting my PC and loading up my dev environment again (no code or configuration changes).
I managed to resolve the issue after a couple of attempts at starting the project (in Visual Studio) by also browsing to the the app URL Microsoft Edge. It loaded find. I then did a hard refresh (Ctrl+F5) in Chrome and it came good too. 
Note: we manually start and leave running Webpack in separate command window, but start the actual application from withing Visual Studio.
My guess is Webpack crapped out behind the scenes and needed some time and a hard refresh in the browser to get the correctly compiled code to the browser again.
